robottelo/ui/base.py

```diff
@@ -47,41 +49,39 @@ def find_element(self, locator):
                 return _webelement
             else:
                 return None
-        except NoSuchElementException as e:
-            logging.debug("%s: Could not locate element %s.",
-                          type(e).__name__,
-                          locator[1])
-            return None
-        except Exception, error:
-            logging.debug("Failed to locate element. ERROR: %s", str(error))
+        except NoSuchElementException as err:
```

Comment:
I think we also need to handle `TimeoutException` here specifically if we are removing the generic Exception from here.
`find_element` function encounters this issue often.